packages/architectura/README.md

```diff
@@ -0,0 +1,141 @@
+# Architectura
+
+Architectura is a light-weight framework using no external dependency.
+It facilitates the implementation of a Domain Driven Design software architecture.
+It provides a simple development API and blazing fast performances.
+
+## Forewords - Read first
+
+Architectura is an opinionated, non-intrusive framework.
+
+### Opinionated
+
+Architectura is opinionated because it follows and incites users to follow a Domain Drive Design software architecture.
+It is also opinionated because it is dependency-free (outside of the Vitruvius Labs ecosystem) and enforces extremely strict typing.
+The opinion of the team behind Architectura is top provide a framework with premium quality standards.
+Architectura also follows the SOLID principles; however, we do not believe that dependency injection is the appropriate answer to the Dependency-inversion principle. To that end, Architectura does not embark any dependency-injection tool.
```

Comment:
`Dependency Inversion Principle` or `dependency inversion principle` instead of `Dependency-inversion principle`

Reply:
In general, I've never seen dependency inversion being hyphenated.